oresat_dxwifi/resources/temperature.py

```diff
@@ -39,11 +44,11 @@ def on_read(self, index: int, subindex: int) -> float:
         """Returns temperature value
 
         Args:
-            index (int): Index in Object Dictionary
-            subindex (int): Subindex of "index" argument
+            index: Index in Object Dictionary
+            subindex: Subindex of "index" argument
 
         Returns:
-            float: Calculated temperature value
+            ret: Calculated temperature value
```

Comment:
We don't need the name of the variable in the `Returns:` section of the docstring. In fact, in this case we don't need the `Returns:` section at all: https://google.github.io/styleguide/pyguide.html#doc-function-returns:

> It may also be omitted if the docstring starts with Returns or Yields (e.g. `"""Returns row from Bigtable as a tuple of strings."""`) and the opening sentence is sufficient to describe the return value.